velox/functions/prestosql/tests/TimestampWithTimeZoneCastTest.cpp

```diff
@@ -51,20 +50,27 @@ class TimestampWithTimeZoneCastTest : public functions::test::CastBaseTest {
 TEST_F(TimestampWithTimeZoneCastTest, fromTimestamp) {
   const auto tsVector = makeNullableFlatVector<Timestamp>(
       {Timestamp(1996, 0), std::nullopt, Timestamp(19920, 0)});
+  auto timestamps =
+      std::vector<int64_t>{1996 * kMillisInSecond, 0, 19920 * kMillisInSecond};
+  auto timezones = std::vector<int16_t>{0, 0, 0};
   const auto expected = makeTimestampWithTimeZoneVector(
-      makeFlatVector<int64_t>(
-          {1996 * kMillisInSecond, 0, 19920 * kMillisInSecond}),
-      makeFlatVector<int16_t>({0, 0, 0}));
+      timestamps.size(),
+      [&](int32_t i) { return timestamps[i]; },
```

Comment:
Let's use the exact types. `int64_t` and `TimeZoneKey`.

Reply:
Refactored. Thank you @majetideepak deepak.